components/sidebar/sidebar_channel/sidebar_channel_link/__snapshots__/sidebar_channel_link.test.tsx.snap

```diff
@@ -39,6 +39,26 @@ exports[`components/sidebar/sidebar_channel/sidebar_channel_link should match sn
       channel_label
     </span>
   </div>
+  <Connect(Component)
```

Comment:
I'm not sure if this is something we can fix easily, but shouldn't this have a component's display name in it? If it's the pencil icon, I hope we don't need to specify a display name for it manually.

Reply:
Hm, my opinion on snapshot testing is to use `react-test-renderer` as jest proposes https://jestjs.io/docs/snapshot-testing#snapshot-testing-with-jest and always mount the component.
Then we could just mock components we don't want to include in the snapshot so that they render just their children or a placeholder.

Snapshot testing via enzyme is weird on the output.

Reply:
At some point, we should do a big discussion on testing to figure out what we want to do for it since I'm personally not even a big fan of snapshot testing in general. I prefer either targeted assertions like "this button renders when this is true" or E2E tests since I find snapshots tend to get ignored because people just `npm test -u` them